YMChat/src/main/java/com/yellowmessenger/ymchat/YMConfig.java

```diff
@@ -25,6 +26,7 @@ public class YMConfig {
     public boolean disableActionsOnLoad = false;
     public boolean useLiteVersion = false;
     public boolean alwaysReload = false;
+    public YMEnableSpeechConfig enableSpeechConfig = new YMEnableSpeechConfig();
```

Comment:
I feel `speechConfig` sounds better than `enableSpeechConfig`. what you think?

Reply:
I kept this because this will be applicable only if enableSpeech is set to true.